content/yaml-quick-start/relocating from AppCenter to Codemagic.md

```diff
@@ -0,0 +1,717 @@
+---
+title: Seamless migration from App Center to Codemagic
+description: How to ship your workflows to Codemagic
+weight: 1
+---
+
+## Quick comparison
+
+
+| **Features**                | **App Center** | **Codemagic**  |
+|-----------------------------| ---------------|----------------|
+| `Swift/Objective-C iOS`     |    ☑️           |      ✅        |
+| `Android apps`              |    ☑️           |      ✅        |
+| `React Native apps`         |    ☑️           |      ✅        |
+| `Flutter apps`              |    ❌          |      ✅        |
+| `Unity apps`                |    ☑️           |      ✅        |
+| `Ionic cAPA`                |    ☑️           |      ✅        |
+| `White-labeling solution`   |    ❌          |      ✅        |
+| `Automatic iOS code signing`|    ❌          |      ✅        |
+| `Manual iOS code signing`   |    ☑️           |      ✅        |
+| `Android code signing`      |    ☑️           |      ✅        |
+| `Automatic build versioning`|    ☑️           |      ✅        |
+| `Running integrations tests`|    ☑️           |      ✅        |
+| `Running unit tests`        |    ☑️           |      ✅        |
+| `App Store publishing`      |    ☑️           |      ✅        |
+| `Play Store publishing`     |    ☑️           |      ✅        |
+| `Firebase App Distribution `|    ❌          |      ✅        |
+| `Slack integration`         |    ❌          |      ✅        |
+| `Email notifications`       |    ☑️           |      ✅        |
+| `macOS M2 support`          |    ❌           |      ✅        |
+| `Linux machines support`    |    ❌           |      ✅        |
+| `Windows machines support`  |    ☑️            |      ✅        |
+| `Over-the-air-update`       |    ☑️            |      ❌*       |
+| `Analytics`                 |    ☑️            |      ❌*       |
```

Comment:
I'd say this two are ☑️ on Codemagic side

Reply:
Would not it be confusing if we use  ☑️ ? because  ☑️ is used for available App Center features. I agree that using ❌ does not look right because we offer alternatives, so these features are still available at some point. Probably, let's use another checkbox e.g ✔️ for features that have alternatives. wdty ?